api-lib/event_triggers/createContributionInteractionEvent.ts

```diff
@@ -15,19 +15,29 @@ export default async function handler(req: VercelRequest, res: VercelResponse) {
             created_at,
             created_with_api_key_hash,
             id,
+            private_stream,
+            big_question_id,
+            profile_id,
           },
         },
       },
     }: EventTriggerPayload<'contributions', 'INSERT'> = req.body;
 
     await mutations.insertInteractionEvents({
-      event_type: 'contribution_create',
+      event_type: private_stream
+        ? 'post_create'
+        : big_question_id
+        ? 'big_question_response_create'
+        : 'contribution_create',
       circle_id: circle_id,
-      profile_id: user_id,
+      profile_id: private_stream || big_question_id ? profile_id : user_id,
       data: {
+        colinks: private_stream || big_question_id ? true : false,
```

Comment:
in addition to just `colinks` as a bool, why dont you also pass `private_stream` and `big_question_id` as fields.

Reply:
done
![image](https://github.com/coordinape/coordinape/assets/34943689/ba49a906-d4cd-445f-a45b-15e4c13cc580)

![image](https://github.com/coordinape/coordinape/assets/34943689/b9303e32-db0d-4396-90da-6ea05b7e0d48)

Reply:
I removed colinks shoul I readd it?
@crabsinger